charts/spire/templates/server/configuration.yaml

```diff
@@ -161,19 +139,22 @@ data:
       sleep 3
     done
 
-    if /opt/spire/bin/spire-server entry show | grep -q "Found 0"; then
-      create \
-        -node \
-        -spiffeID spiffe://{{ .Values.trustDomain }}/cluster \
-        -selector k8s_sat:cluster:{{ .Values.trustDomain }} \
+    spiffe_id="spiffe://{{ .Values.trustDomain }}/cluster"
+    selectors="-selector k8s_sat:cluster:{{ .Values.trustDomain }}"
+    if ! /opt/spire/bin/spire-server entry show -spiffeID "${spiffe_id}" ${selectors} | grep -q -w "${spiffe_id}"; then
+      /opt/spire/bin/spire-server entry create -node -spiffeID "${spiffe_id}" ${selectors}
+    else
+      echo "Entry already exists: ${spiffe_id}"
+    fi
 
-      create \
+    spiffe_id="spiffe://{{ .Values.trustDomain }}/spire-jwks"
+    selectors="-selector k8s:ns:{{ .Release.Namespace }} -selector k8s:sa:{{ include "spire.fullname" . }}-jwks -selector k8s:pod-label:app.kubernetes.io/name:{{ include "spire.fullname" . }}-jwks -selector k8s:container-name:jwks"
```

Comment:
This is more difficult to read and change, Why make this one line.

Reply:
Added line breaks.